returnn/util/file_cache.py

```diff
@@ -312,6 +326,26 @@ def _check_existing_copied_file_maybe_cleanup(src_filename: str, dst_filename: s
         return True
 
 
+def _copy_with_prealloc(src: str, dst: str):
+    """
+    copies from src to dst preallocating the disk space at dst before copying
+
+    :param src: source file
+    :param dst: destination file
```

Comment:
I don't like such redundant docstrings. If you think the variable name is not self-explanatory, why not just change the variable names?

Reply:
No honestly I added this to keep the linter happy. I don't usually work in PyCharm so I don't get the pyCharm lints for free. I've since removed them.

Reply:
There is a check that all public functions/methods/classes/modules must have a docstring. But:

* It does not apply for private functions. Those don't need to have any docstring.
* It does not mean that args must be documented.

However, when you document args, then it is a warning when you only document some of them but not all of them, or when you document some non-existing one.